core/chains/evm/client/errors.go

```diff
@@ -108,7 +108,36 @@ var avalanche = ClientErrors{
 	NonceTooLow: regexp.MustCompile(`(: |^)nonce too low: address 0x[0-9a-fA-F]{40} current nonce \([\d]+\) > tx nonce \([\d]+\)$`),
 }
 
-var clients = []ClientErrors{parity, geth, arbitrum, optimism, substrate, avalanche}
+// Nethermind
+// All errors: https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.TxPool/AcceptTxResult.cs
+// All filters: https://github.com/NethermindEth/nethermind/tree/9b68ec048c65f4b44fb863164c0dec3f7780d820/src/Nethermind/Nethermind.TxPool/Filters
+var nethermindFatal = regexp.MustCompile(`(: |^)(SenderIsContract|Invalid|Int256Overflow|FailedToResolveSender)`)
+
+// Int256Overflow: Calculation of gas price * gas limit + value overflowed int256.
+// though it feels like Int256Overflow could be use for high fees as well?
+var nethermind = ClientErrors{
+	// Assuming all nonce errors are due to the nonce being too low
+	// NonceGap: The nonce is not the next nonce after the last nonce of this sender present in TxPool.
+	// OldNonce: The EOA (externally owned account) that signed this transaction (sender) has already signed and executed a transaction with the same nonce.
+	NonceTooLow: regexp.MustCompile(`(: |^)(OldNonce|NonceGap)`),
+	FeeTooLow:   regexp.MustCompile(`(: |^)(FeeTooLow|FeeTooLowToCompete)`),
+
+	// GasLimitExceeded: Transaction gas limit exceeds the block gas limit.
+	FeeTooHigh: regexp.MustCompile(`(: |^)GasLimitExceeded`),
```

Comment:
That isn't a FeeTooHigh, that's a fatal error.

Reply:
understood, changing it.